CONTRIBUTING.md

```diff
@@ -1,10 +1,10 @@
-# Contributing to Eclipse Project for WebSocket
+# Contributing to Jakarta WebSocket Project
```

Comment:
I think most projects just said here something akin to "Contributing to Jakarta WebSocket", e.g. leaving out the "Project" bit.

Reply:
Will do.

Reply:
Done.